<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QTreeView" name="treeViewPath">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.523000000000000" starty="0.119000000000000" endx="0.523000000000000" endy="0.870000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>66</red>
                <green>96</green>
                <blue>122</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>3</red>
                <green>65</green>
                <blue>99</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Light">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Midlight">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>212</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Dark">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>85</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Mid">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>113</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="BrightText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.523000000000000" starty="0.119000000000000" endx="0.523000000000000" endy="0.870000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>66</red>
                <green>96</green>
                <blue>122</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>3</red>
                <green>65</green>
                <blue>99</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.523000000000000" starty="0.119000000000000" endx="0.523000000000000" endy="0.870000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>66</red>
                <green>96</green>
                <blue>122</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>3</red>
                <green>65</green>
                <blue>99</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Shadow">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="AlternateBase">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>127</red>
              <green>212</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ToolTipBase">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>220</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ToolTipText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.523000000000000" starty="0.119000000000000" endx="0.523000000000000" endy="0.870000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>66</red>
                <green>96</green>
                <blue>122</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>3</red>
                <green>65</green>
                <blue>99</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Light">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Midlight">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>212</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Dark">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>85</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Mid">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>113</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="BrightText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.523000000000000" starty="0.119000000000000" endx="0.523000000000000" endy="0.870000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>66</red>
                <green>96</green>
                <blue>122</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>3</red>
                <green>65</green>
                <blue>99</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.523000000000000" starty="0.119000000000000" endx="0.523000000000000" endy="0.870000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>66</red>
                <green>96</green>
                <blue>122</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>3</red>
                <green>65</green>
                <blue>99</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Shadow">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="AlternateBase">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>127</red>
              <green>212</green>
              <blue>127</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ToolTipBase">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>220</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ToolTipText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Button">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.523000000000000" starty="0.119000000000000" endx="0.523000000000000" endy="0.870000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>66</red>
                <green>96</green>
                <blue>122</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>3</red>
                <green>65</green>
                <blue>99</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Light">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Midlight">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>212</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Dark">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>85</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Mid">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>113</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="BrightText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.523000000000000" starty="0.119000000000000" endx="0.523000000000000" endy="0.870000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>66</red>
                <green>96</green>
                <blue>122</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>3</red>
                <green>65</green>
                <blue>99</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="LinearGradientPattern">
             <gradient startx="0.523000000000000" starty="0.119000000000000" endx="0.523000000000000" endy="0.870000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
              <gradientstop position="0.000000000000000">
               <color alpha="255">
                <red>66</red>
                <green>96</green>
                <blue>122</blue>
               </color>
              </gradientstop>
              <gradientstop position="1.000000000000000">
               <color alpha="255">
                <red>3</red>
                <green>65</green>
                <blue>99</blue>
               </color>
              </gradientstop>
             </gradient>
            </brush>
           </colorrole>
           <colorrole role="Shadow">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="AlternateBase">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>170</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ToolTipBase">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>220</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ToolTipText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0.523, y1:0.119, x2:0.523, y2:0.87, stop:0 rgba(66, 96, 122, 255), stop:1 rgba(3, 65, 99, 255));
color: rgb(255, 255, 255);
background-image: url(:/images/strip.png);</string>
        </property>
        <property name="lineWidth">
         <number>9</number>
        </property>
        <property name="midLineWidth">
         <number>16</number>
        </property>
        <property name="autoScrollMargin">
         <number>21</number>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DragDrop</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="itemsExpandable">
         <bool>true</bool>
        </property>
        <property name="animated">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="btnSearch">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ldtFilter"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QComboBox" name="cbbiconSize">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbbiconList">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbbiconContentList">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGraphicsView" name="graphicsView">
            <property name="styleSheet">
             <string notr="true">background-color: qlineargradient(spread:pad, x1:0.523, y1:0.119, x2:0.523, y2:0.87, stop:0 rgba(66, 96, 122, 255), stop:1 rgba(3, 65, 99, 255));
color: rgb(255, 255, 255);
background-image: url(:/images/strip.png);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1105</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMerge">
   <property name="text">
    <string>Merge ...</string>
   </property>
   <property name="toolTip">
    <string>merge file max</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy ...</string>
   </property>
   <property name="toolTip">
    <string>Copy file to 3dsMax</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open ...</string>
   </property>
   <property name="toolTip">
    <string>Open asset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete...</string>
   </property>
  </action>
  <action name="actionAssign_to_Shader">
   <property name="text">
    <string>Assign to Shader</string>
   </property>
  </action>
  <action name="actionExecute">
   <property name="text">
    <string>Execute</string>
   </property>
  </action>
  <action name="actionUp">
   <property name="text">
    <string>Up</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
